testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -189,19 +195,13 @@ async fn wait_node_stateful_set(
     .await
 }
 
+// TODO: set validator image tag by kube api rather than helm
 pub fn set_validator_image_tag(
```

Comment:
Can you explain why did this change?

Reply:
This changed 1.5 months ago when we rewrote both our testnet setup and most of Forge k8s. This functionality was never verified and the code was never deleted since we'll likely have to use it again (hence the TODO).

The tl;dr is that we're using a single helm release to manage all nodes nowadays, whereas in the old setup we used N helm releases for N nodes. The former is more scalable, but this means we will need to rely on interacting more with the `kube` API directly rather than simply invoking `helm` commands.